test/one/profiler/test/TestProcess.java

```diff
@@ -248,14 +248,16 @@ public Output profile(String args) throws IOException, TimeoutException, Interru
 
     public Output profile(String args, boolean sudo) throws IOException, TimeoutException, InterruptedException {
         List<String> cmd = new ArrayList<>();
-        if (sudo) {
+        System.err.println("Run profile with");
+	if (sudo) {
+		System.err.println("    sudo");
             cmd.add("/usr/bin/sudo");
         }
         cmd.add("build/bin/asprof");
         addArgs(cmd, args);
         cmd.add(Long.toString(pid()));
         log.log(Level.FINE, "Profiling " + cmd);
-
+System.out.println("   command " + cmd);
```

Comment:
Is it some debugging leftover?
A command is already logged in the above statement.